c/CasADi/CasADi-3.7.0-gfbf-2024a.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'PythonBundle'
+
+name = 'CasADi'
+version = '3.7.0'
+
+homepage = 'https://web.casadi.org'
+description = """CasADi is an open-source tool for nonlinear optimization and algorithmic differentiation.
+It facilitates rapid — yet efficient — implementation of different methods for numerical optimal control,
+both in an offline context and for nonlinear model predictive control (NMPC)."""
+
+toolchain = {'name': 'gfbf', 'version': '2024a'}
+dependencies = [('Python', '3.12.3'),
+                ('SciPy-bundle', '2024.05'),
+                ('CMake', '3.29.3')]
+
+exts_list = [
+    ('casadi', version, {
+        'modulename': 'casadi',
+        'checksums': ['21254f17eb5551c4a938641cc1b815ff3da27271ab2c36e44a3e90ec50ba471f'],
+    }),
+]
+
+sanity_check_paths = {
```

Comment:
Also for `PythonBundle` kind of easyblocks, the check on the `lib` directory happens automatically. So, this can go either.